src/components/SQFormDialog/SQFormDialogInner.js

```diff
@@ -124,17 +127,57 @@ function SQFormDialogInner({
               showSecondaryButton ? actionsClasses : primaryActionsClasses
             }
           >
-            {showSecondaryButton && (
-              <RoundedButton
-                title={cancelButtonText}
-                onClick={handleCancel}
-                color="secondary"
-                variant="outlined"
+            {showTertiaryButton ? (
```

Comment:
We can take a functional approach to avoid these conditionals.
We can create render functions that contain the logic, we can create separate components that handle this logic and except the necessary props, or we can live this as is.

Reply:
I'm in the "Create render functions" camp on this one. I found it a little hard to follow what was going on in the diff.